storybook/stories/API/cartesian/Line.stories.tsx

```diff
@@ -2,59 +2,121 @@
 import React from 'react';
 import { within } from '@storybook/testing-library';
 import { expect } from '@storybook/jest';
-import { Surface, Line, ResponsiveContainer } from '../../../../src';
-import { coordinateWithValueData } from '../../data';
+import { Surface, Line, ResponsiveContainer, ComposedChart } from '../../../../src';
+import { ccoordinateWithNullY, coordinateData, coordinateWithValueData } from '../../data';
 
 export default {
   component: Line,
   argTypes: {
+    connectNulls: {
+      control: {
+        type: 'boolean',
+      },
+    },
     stroke: {
       control: { type: 'color' },
     },
     fill: {
       control: { type: 'color' },
     },
+    type: {
+      // TODO: These options should be generated from the type directly instead of duplicating the type information here. Will iterate.
+      options: [
+        'basis',
+        'basisClosed',
+        'basisOpen',
+        'linear',
+        'linearClosed',
+        'natural',
+        'monotoneX',
+        'monotoneY',
+        'monotone',
+        'step',
+        'stepBefore',
+        'stepAfter',
+      ],
+      control: {
+        type: 'select',
+      },
+    },
   },
 };
 
 export const Simple = {
   render: (args: Record<string, any>) => {
-    const { data, ...lineArgs } = args;
-
-    const [surfaceWidth, surfaceHeight] = [600, 300];
+    const [height, width] = [300, 300];
 
+    const { data, ...lineArgs } = args;
     return (
-      <ResponsiveContainer width="100%" height={surfaceHeight}>
-        <Surface
-          width={surfaceWidth}
-          height={surfaceHeight}
-          viewBox={{
-            x: 0,
-            y: 0,
-            width: surfaceWidth,
-            height: surfaceHeight,
+      <ResponsiveContainer width="100%" height={height}>
+        <ComposedChart
+          width={width}
+          height={height}
+          margin={{
+            top: 20,
+            right: 20,
+            bottom: 20,
+            left: 20,
           }}
+          data={data}
         >
-          <Line dataKey="value" isAnimationActive={false} points={data} {...lineArgs} />
-        </Surface>
+          {/* Setting a default for the dataKey here, allows us to hide the dataKey
+          as a control in other stories, yet reusing the same template. */}
+          <Line isAnimationActive={false} dataKey="y" {...lineArgs} />
+        </ComposedChart>
       </ResponsiveContainer>
     );
   },
   args: {
     data: coordinateWithValueData,
+    dataKey: 'y',
+  },
+  parameters: {
+    controls: { include: ['data', 'dataKey'] },
+    docs: {
+      description: {
+        story: 'The dataKey defines the y-Values of a Line. Without an xAxis, the index is used for x',
```

Comment:
Let's amend the punctuation here for consistency.

```suggestion
        story: 'The dataKey defines the y-Values of a Line. Without an xAxis, the index is used for x.',
```